include/boost/beast/http/impl/fields.hpp

```diff
@@ -953,18 +947,22 @@ set_keep_alive_impl(
 template<class Allocator>
 auto
 basic_fields<Allocator>::
-new_element(field name,
-    string_view sname, string_view value) ->
-        element&
-{
-    if(sname.size() + 2 >
-            (std::numeric_limits<off_t>::max)())
-        BOOST_THROW_EXCEPTION(std::length_error{
-            "field name too large"});
-    if(value.size() + 2 >
-            (std::numeric_limits<off_t>::max)())
-        BOOST_THROW_EXCEPTION(std::length_error{
-            "field value too large"});
+new_element(
+    field name,
+    string_view sname,
```

Comment:
`field name` was a mistake on my part. It should have been `field id`. And then `sname` could just be `name` and we wouldn't have two confusingly similar variables.